Analyzer/interface/Regions.h

```diff
@@ -10,10 +10,9 @@
 #include "TDirectory.h"
 
 
-TRandom3* RNG = new TRandom3();
 
-float K = 1;
-float C = 1;
+float K = 2.30;
+float C = 3.17;
```

Comment:
Why does this matter? It should be not hardcoded by modified from the config, right?

Reply:
It should, forget of my side